### PR TITLE
[SimpleS3] Dont use presignUrl, use parent::getEndpoint()

### PR DESCRIPTION
**Pull request by Nyholm**
Blocked by #630

### CODE REVIEW

File: src/Integration/Aws/SimpleS3/src/SimpleS3Client.php [hunk 1]
```diff
@@ -17,20 +17,7 @@ class SimpleS3Client extends S3Client
...
-        $url = $this->presign($input, new \DateTimeImmutable('+10minutes'));
-
-        // remove all query parameters.
-        if (false === $pos = strpos($url, '?')) {
-            // If the client didn't have any credentials, there will not be any query string
-            return $url;
-        }
-
-        return substr($url, 0, $pos);
+        return $this->getEndpoint(sprintf('/%s/%s', $bucket, $key), [], null);
```
Comment (jderusse) [inline src/Integration/Aws/SimpleS3/src/SimpleS3Client.php:20]: key and buck should be `urlencod`

Comment (jderusse) [reply src/Integration/Aws/SimpleS3/src/SimpleS3Client.php:20]: hmm. and `/` in key should be conserved. The right uri is:
`$uri = '/' . rawurlencode($bucket) . '/' . str_replace('%2F', '/', rawurlencode($key));`

Comment (Nyholm) [reply src/Integration/Aws/SimpleS3/src/SimpleS3Client.php:20]: Yeah, You are correct. THank you

**Comment by Nyholm** [comment]
Yeah, Indeed. 

Thank you for the review

**Pull request merged**